build/build_standalone3.less

```diff
@@ -21,6 +21,8 @@
 @component-active-bg:           @brand-primary;
 @dropdown-link-active-bg:       @component-active-bg;
 @dropdown-link-hover-bg:        @dropdown-link-active-bg;
+@dropdown-fallback-border:      @dropdown-fallback-border;
+@dropdown-bg:                   @dropdown-bg;
```

Comment:
You are right, these are not necessary i removed them and now i am using the correct variable for the arrow border (`@dropdown-border`) https://github.com/eternicode/bootstrap-datepicker/commit/388f192ebde778837484c1521b91a087a04182f1